components/brave_vpn/browser/connection/win/brave_vpn_helper/BUILD.gn

```diff
@@ -28,6 +29,14 @@ source_set("common") {
     "//brave/components/brave_vpn/common",
     "//chrome/install_static:install_static_util",
   ]
+  defines = []
+  if (brave_channel == "nightly") {
+    defines += [ "CHANNEL_NIGHTLY" ]
+  } else if (brave_channel == "beta") {
+    defines += [ "CHANNEL_BETA" ]
+  } else {
```

Comment:
Curious why `CHANNEL_DEV` is used for default?
With this, `dev` release channel and `stable` release channel will use same guid.

Reply:
it was too early to review at that time, it is committed now